lib/events/filesessions/fileasync.go

```diff
@@ -386,13 +387,36 @@ func (u *Uploader) uploadEncryptedRecording(ctx context.Context, sessionID strin
 	}
 
 	partIter := func(yield func([]byte, error) bool) {
+		// These upload parts are each ~128KiB. Usually these parts are consumed and reconstructed by Auth in 5MiB
```

Comment:
We probably don't need to decide for this PR, but we should probably reevaluate this size at least for encrypted recordings. Since each part will equate to a decryption call to whatever keystore is in use, individual part size will have a direct impact on keystore pressure.